### PR TITLE
Se agrega la barra de navegacion inferior para "mis plantas" screen

### PR DESCRIPTION
**Pull request by devdevsamuel**

### CODE REVIEW

**Comment by devdevsamuel** [comment]
Screen Shot 2022-11-24 at 1 32 58 AM

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,79 @@
...
+import React from "react";
+import { useState } from "react";
+import { Container } from "@mui/material"
+
+//import { BottomNavigation } from './components/BottomNavigation';
+//import { CheckPlantButton } from './components/CheckPlantButton';
+
+// import { CreatePlantButton } from "./components/CreatePlantButton";
+// import { PlantItem } from "./components/PlantItem";
+// import { PlantList } from "./components/PlantList";
```
Comment (calderonsteven) [inline src/App.js:10]: Please remove commented code

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,79 @@
...
+import { PlantItem } from "./components/PlantItem.js";
+
+
+
+
+const plants=[
+  {text:'cebolla', completed:false},
+ 
+  {text:'Llorar ', completed:false}
+];
```
Comment (calderonsteven) [inline src/App.js:27]: please format/indent the whole file so it looks better

File: src/components/BottomNavigationMUI.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import { BottomNavigation, BottomNavigationAction } from "@mui/material";
+import HomeIcon from "@mui/icons-material/Home";
+import AddIcon from '@mui/icons-material/Add';
+
+import { useState } from "react";
+import "./BottomNavigationMUI.css";
+// import Home from "@mui/icons-material/Home";
```
Comment (calderonsteven) [inline src/components/BottomNavigationMUI.js:9]: remove commented code

**Review by calderonsteven — CHANGES_REQUESTED** [review]
Some small changes to address.

**Pull request merged**